src/main/java/frc/robot/Constants.java

```diff
@@ -39,9 +42,9 @@
     public static final int COUNTS_PER_REVOLUTION = 42;
 
 
-	// Xbox Controller constants
-	public static final int XBOX_CONTROLLER = 0;
-	public static final int DRIVE_FWD_REV = XboxControllerButtons.LSTICK_Y_AXIS;
-    public static final int DRIVE_LEFT_RIGHT = XboxControllerButtons.RSTICK_X_AXIS;
+    // Xbox Controller constants
+	public static final int XBOX_CONTROLLER = 0; 
+	public static final int DRIVE_FWD_REV = XboxController.Axis.kLeftY.value;
+    public static final int DRIVE_LEFT_RIGHT = XboxController.Axis.kRightX.value;
     public static final int REVERSE_DRIVE_DIRECTION = XboxControllerButtons.BUTTON_LSTICK;
```

Comment:
You can use the [`XboxController.Button`](https://first.wpi.edu/FRC/roborio/release/docs/java/edu/wpi/first/wpilibj/XboxController.Button.html) enum for this.

Reply:
Tyler, I am not too sure as to why we are using a button rather than the joystick for movement, or do buttons function as a joystick too?

Reply:
What do you mean using a button instead of a joystick for movement? Are you talking about `REVERSE_DRIVE_DIRECTION`? That's not for movement, that's the button that when pressed, reverses the driving direction.